### ISSUE TITLE
webpack 2 resolves to es6, requires further transpilation

### ISSUE THREAD

**Issue by ragnarok56** [vkbansal/react-contextmenu#77]
## Version
2.2.0

Using webpack 2 

webpack.config.babel.js
```
        ...
        entry: {
            main: './index.js'
        },
        ...
        module: {
            rules: [
                {
                    test: /\.jsx?$/,
                    exclude: /node_modules/,
                    use: [
                        {
                            loader: "babel-loader",
                            options: {
                                babelrc: false,
                                presets: [
                                    ['es2015', { 'modules': false }],
                                    'stage-3',
                                    'react'
                                ]
                            }
                        }
                    ]
                }
            ]
        }
        ...
```

index.js
```
import { ContextMenu } from 'react-contextmenu'
```

```
> webpack -p --display-error-details --env.production

Hash: bbbbe622a962daacd815
Version: webpack 2.2.1
Time: 397ms
                       Asset    Size  Chunks             Chunk Names
main-bbbbe622a962daacd815.js  134 kB       0  [emitted]  main
   [1] ./~/object-assign/index.js 2.11 kB {0} [built]
   [2] ./~/react-contextmenu/es6/helpers.js 783 bytes {0} [built]
   [3] ./~/react/lib/ReactElement.js 11.2 kB {0} [built]
   [6] ./~/classnames/index.js 1.1 kB {0} [built]
   [7] ./~/react-contextmenu/es6/actions.js 1.01 kB {0} [built]
   [8] ./~/react/react.js 56 bytes {0} [built]
  [18] ./~/react-contextmenu/es6/index.js 223 bytes {0} [built]
  [19] ./~/react-contextmenu/es6/ContextMenu.js 4.72 kB {0} [built]
  [20] ./~/react-contextmenu/es6/ContextMenuTrigger.js 2.79 kB {0} [built]
  [21] ./~/react-contextmenu/es6/MenuItem.js 1.68 kB {0} [built]
  [22] ./~/react-contextmenu/es6/SubMenu.js 5.21 kB {0} [built]
  [23] ./~/react-contextmenu/es6/globalEventListener.js 1.21 kB {0} [built]
  [26] ./~/react/lib/React.js 2.69 kB {0} [built]
  [28] ./~/react/lib/ReactClass.js 26.5 kB {0} [built]
  [36] ./index.js 48 bytes {0} [built]
    + 22 hidden modules

ERROR in main-bbbbe622a962daacd815.js from UglifyJs
SyntaxError: Name expected [main-bbbbe622a962daacd815.js:246,28]
```

For importing dependencies, webpack 2 uses [mainfields](https://webpack.js.org/configuration/resolve/#resolve-mainfields) to determine what to load.  The  default precedence is 'browser', then 'module', then 'main'.  Since there is no 'browser' field, it will load the non-transpiled es6 version, but doesn't further transpile things down, so when its passed to uglifyjs, it fails.

**Comment by Kimahriman** [comment]
The best justification for this can be found [here](https://github.com/rollup/rollup/wiki/pkg.module): "pkg.module will point to a module that has ES2015 module syntax but otherwise only syntax features that node supports."

"If you're using things like arrow functions and classes (or more exotic features like decorators and object spread), you should transpile them if you want your library to work in environments that don't support those features. Otherwise, whoever uses your library will have to transpile it themselves, including configuring their transpiler to handle your code, which might involve esoteric transforms and conflicting versions of Babel."

**Comment by vkbansal** [comment]
My bad. I had wrong understanding of how the "module" works. I was under the impression that people will manually transpile es6 modules. I was wrong. Will fix it ASAP.

**Comment by vkbansal** [comment]
I have published v2.2.1 with the fix. Please reopen the issue if you still face the problem.

**Comment by ragnarok56** [comment]
awesome, that fixes it.  thanks!  we spent some time trying to see if it was on our end, since we just updated to webpack 2 today.  it's not really spelled out how to setup `module`, so completely understandable
